opentelemetry-sdk/src/opentelemetry/sdk/metrics/__init__.py

```diff
@@ -141,6 +176,7 @@ def __init__(
         description: str,
         unit: str,
         value_type: Type[metrics_api.ValueT],
+        meter: "Meter",
```

Comment:
Is the only reason to include the meter here to prevent using labelsets created for other meter instances? As I understand the spec, we don't want to guarantee that labelsets created via one implementation can be used in another (i.e. they're defined in the SDK, not the API), but that's seems like a very different concern.

Reply:
Note that the go prototype is doing this: https://github.com/open-telemetry/opentelemetry-go/blob/e6d725626d4629220a2de0112570adf80d50be21/sdk/metric/sdk.go#L283. @c24t to take this up in specs.